### PR TITLE
Update "Current Members of Core" section

### PR DESCRIPTION
**Pull request by marcelotrevisani**

### CODE REVIEW

**Comment by jjhelmus** [comment]
A few of the member on this list are emeritus.  Should we split this list to show those member under a separate heading?  
The [core.csv](https://github.com/conda-forge/conda-forge.github.io/blob/master/src/core.csv) and [emeritus.csv](https://github.com/conda-forge/conda-forge.github.io/blob/master/src/emeritus.csv files in this repository give list of the current core members and emeritus members.

**Comment by CJ-Wright** [comment]
You might consider using this as well (it has current and emeritus) https://github.com/conda-forge/by-the-numbers/blob/master/plots/current-core-devs.png although it needs to be updated

**Comment by marcelotrevisani** [comment]
@jjhelmus @CJ-Wright 
should we split the list as @jjhelmus mentioned ? 

> You might consider using this as well (it has current and emeritus) https://github.com/conda-forge/by-the-numbers/blob/master/plots/current-core-devs.png although it needs to be updated

I will do that as well, thanks

**Comment by marcelotrevisani** [comment]
I believe all the core and emeritus members are in this PR
Did I miss someone?

**Comment by CJ-Wright** [comment]
At some point we might consider just jinja templating this.

File: src/orga/governance.rst [hunk 1]
```diff
@@ -309,27 +309,37 @@ Current Members of Core
...
 * Mike Sarahan, @msarahan
-* Anthony Scopatz, @scopatz
-* Patrick Snape, @patricksnape
+* Min Ragan-Kelley, @minrk
 * Patrick Sodré, @sodre
-* Dougal J. Sutherland, @dougalsutherland
 * Peter K. G. Williams, @pkgw
-* Christopher J. "CJ" Wright, @CJ-Wright
+* Uwe L. Korn, @xhochy
+* Wolf Vollprecht, @wolfv
```
Comment (scopatz) [inline src/orga/governance.rst:332]: Isn't alphabetical order usually by last name?

Comment (marcelotrevisani) [reply src/orga/governance.rst:332]: I think if the name was written as ``Surname, the first name`` I would agree with you
but I think as we wrote the name in full that might be a bit odd (at least for me) to sort it looking for the last name and writing the first name at the beginning

File: src/core.csv [hunk 2]
```diff
@@ -1,3 +1,4 @@
+github_username,email,name
```
Comment (CJ-Wright) [inline src/core.csv:1]: I'm not certain if this will play well with the voting system.

Comment (marcelotrevisani) [reply src/core.csv:1]: How can we check it?

Comment (CJ-Wright) [reply src/core.csv:1]: You could try to make a dummy vote in polys

Comment (marcelotrevisani) [reply src/core.csv:1]: I tested and it worked fine

Comment (marcelotrevisani) [reply src/core.csv:1]: I tested with the csv header and add you and me to a pool
it seems to be working fine

**Pull request merged**